plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -65,7 +65,7 @@ import {
   METRIC_DB_CLIENT_OPERATION_DURATION,
   ATTR_DB_NAMESPACE,
   ATTR_DB_OPERATION_NAME,
-} from '@opentelemetry/semantic-conventions/incubating';
+} from './semantic-conventions';
```

Comment:
https://github.com/open-telemetry/opentelemetry-js/issues/5182#issuecomment-2536558790 suggested using `semconv.ts`; I went with `semantic-conventions.ts` since it directly matches the package name. Happy to use the other suggestion if that's strongly preferred.